### DISCUSSION TITLE
Unable to connect my wallet via wallet connect since June 28th

### DISCUSSION THREAD

**Discussion by Eddiemiggs** [WalletConnect/walletconnect-monorepo#2923] [Developer Support] [answered]
As far as I understand I need to upgrade to wallet connect to V2 so I can connect my trust wallet again on iPhone. Anyone have an idea how to go about it??

**Comment by chadyj** [comment]
Can you please let us know what version of Trust you are using (and did you try updating?)
What dapps are you having issues with?

**Comment by Eddiemiggs** [comment]
I’m using Trust wallet version 8.9. I can’t update it because no update is available through apples App Store. Sent from my iPhoneOn Jul 1, 2023, at 9:07 AM, Chad ***@***.***> wrote:﻿
Can you please let us know what version of Trust you are using (and did you try updating?)
What dapps are you having issues with?

**Comment by Eddiemiggs** [comment]
The dapp I’m trying to connect my wallet to is the drip networkSent from my iPhoneOn Jul 1, 2023, at 9:23 AM, Ed Migliaccio ***@***.***> wrote:﻿I’m using Trust wallet version 8.9. I can’t update it because no update is available through apples App Store. Sent from my iPhoneOn Jul 1, 2023, at 9:07 AM, Chad ***@***.***> wrote:﻿
Can you please let us know what version of Trust you are using (and did you try updating?)
What dapps are you having issues with?

**Comment by chadyj** [comment, marked as answer]
Unfortunately Drip Network hasn't yet upgraded to WalletConnect v2. Please try contacting them and asking them to upgrade.

**Comment by Eddiemiggs** [comment]
Would it help to import my wallet to Metamask and connect that way??Sent from my iPhoneOn Jul 1, 2023, at 10:00 AM, Chad ***@***.***> wrote:﻿
Unfortunately Drip Network hasn't yet upgraded to WalletConnect v2. Please try contacting them and asking them to upgrade.
